README.md

```diff
@@ -72,11 +73,12 @@ You can optionally copy the contents of `.env.example` to a `.env` file that wil
 - `LAKEHOUSE_ABFSS_PREFIX`: The prefix for the Azure Blob File System Storage (ABFSS) path.  Should match pattern `abfss://<workspace_id>@msit-onelake.dfs.fabric.microsoft.com/<lakehouse_id>`.  Get this value by selecting "Properties" on your Lakehouse Tables location and copying "ABFS path".
 - `DPS_TABLE_NAME`: The name of the table where data point values and timestamps should be stored in Fabric.  The replicator will create the table if it does not exist.
 - `TS_TABLE_NAME`: The name of the table where time series metadata should be stored in Fabric.  The replicator will create the table if it does not exist.
+- `EVENT_TABLE_NAME`: The name of the table where event data should be stored in Fabric. The replicator will create the table if it does not exist.
 
 ## Fabric Extractor Variables
-- `EXTRACTOR_EVENT_PATH`: The ABFSS file path for the events table in a Fabric lakehouse.
-- `EXTRACTOR_FILE_PATH`: The ABFSS file path for the files in a Fabric lakehouse.
-- `EXTRACTOR_RAW_TS_PATH`: The ABFSS file path for the gittimeseries table in a Fabric lakehouse.
+- `EXTRACTOR_EVENT_PATH`: The table path for the events table in a Fabric lakehouse.  It's the relative path after the ABFSS prefix (e.g. "Tables/RawEvents")
+- `EXTRACTOR_FILE_PATH`: The file path for the files in a Fabric lakehouse. It's the relative path after the ABFSS prefix (e.g. "Files/Tanks.png")
```

Comment:
Is this a file or a directory? The example implies file, the name implies directory.

Reply:
A file not a directory. Which name are you referring?

Reply:
EXTRACTOR_FILE_PATH

If I trace into the code, it looks like it is used here:
```
for file in file_system_client.get_paths(file_path):
```
Which looks like it should be a directory.

Reply:
Also, if it were a file, then we can only specify a single file?

Reply:
Good point. We only manually tested that it works for a single file. Just tested and it also works for a directory. Added it to the doc.